org.sofproject.gst.topo/src/org/sofproject/gst/json/JsonCustomOptionPane.java

```diff
@@ -0,0 +1,156 @@
+package org.sofproject.gst.json;
+import java.awt.*;
+import java.awt.event.*;
+import java.beans.*;
```

Comment:
I'd not use wildcard imports at all

Reply:
IDE quirks maybe?

Reply:
AFAIK IntelliJ and Eclipse will ask which one you want or default to no wildcards.

Reply:
@jajanusz I'm more versed in IntelliJ and I think it didn't ask me but it defaulted to doing wildcards. I know I have changed it to no wildcards when it did that and I have also done quick actions in the various Java projects I've done (college homework really). Not sure about Eclipse.

Reply:
@edominia thanks